src/components/layout/header.js

```diff
@@ -1 +1,12 @@
-/*header will go here */
\ No newline at end of file
+import React from 'react'
+
+const Header = () =>{
```

Comment:
@lovetrivedi Proper indentation is missing.

Reply:
done